sdk/test/NetStandard/IntegrationTests/IntegrationTests/S3/S3ServiceClientTests.cs

```diff
@@ -53,5 +54,54 @@ public async Task TestS3PutWithBodyProperty()
                 Assert.Equal(putObjectRequest.ContentBody, content);
             }
         }
+
+#if NET8_0_OR_GREATER
+        [Fact]
+        public async Task TestDisableCanonicalization()
+        {
+            string key = $"/path/../{Guid.NewGuid().ToString()}.txt";
```

Comment:
Not necessary, but it might be good to add a test with SIGV2. Since SIGV2 follows a different path for GetPresignedUrl, the behavior might be different

Reply:
I ran my forcing SigV2 and it worked correctly. Since both SigV2 and V4 use the same utility method I don't think we need to add another expensive, time wise, integ tests for SigV2.